igneous/task_creation.py

```diff
@@ -184,9 +184,10 @@ def create_deletion_tasks(task_queue, layer_path):
     task_queue.insert(task)
   task_queue.wait('Uploading DeleteTasks')
 
-def create_skeletonizing_tasks(task_queue, cloudpath, mip, shape=Vec(512, 512, 512), teasar_params=[10, 10]):
+def create_skeletonizing_tasks(task_queue, cloudpath, mip, shape=Vec(512, 512, 512),
+                               teasar_params={'scale':10, 'const': 10}, info=None):
   shape = Vec(*shape)
-  vol = CloudVolume(cloudpath, mip=mip)
+  vol = CloudVolume(cloudpath, mip=mip, info=info)
```

Comment:
It's not really standard to add info to these tasks.

Reply:
So the use case I have for this is for the parameter setting, if you launch hundreds of skeleton jobs with different parameters all pointed at the same cloudvolume, then they are going to fight over what name to put in the skeleton field, and end up saving skeletons over themselves, rather than putting them all in seperate folders.  I'd be open to other suggestions, but if i want to test parameters in parallel I need to not be limited to saving skeletons into a folder specified in the info file that is decided before each of the jobs is launched.